notebooks/week2/04_log_and_register_custom_model.py

```
@@ -0,0 +1,123 @@
+with open("model_version.json", "w") as json_file:
+    json.dump(model_version.__dict__, json_file, indent=4)
+# COMMAND ----------
+model_version_alias = "the_bestest_model"
+client.set_registered_model_alias(model_name, model_version_alias, "1")
+
+model_uri = f"models:/{model_name}@{model_version_alias}"
+model = mlflow.pyfunc.load_model(model_uri)
+# COMMAND ----------
+client.get_model_version_by_alias(model_name, model_version_alias)
+# COMMAND ----------
+model
```

Ruff

Found useless expression. Either assign it to a variable or remove it.

(B018)